DevEdu.API/DevEdu.API/Configuration/MapperProfile.cs

```diff
@@ -52,6 +56,25 @@ private void CreateMappingFromDto()
                 .ForMember(dest => dest.Date, opt => opt.MapFrom(src => src.Date.ToString(_dateFormat)));
             CreateMap<CourseTopicDto, CourseTopicOutputModel>();
             CreateMap<UserDto, UserInfoOutputModel>();
+            CreateMap<TagDto, TagOutputModel>();
+            CreateMap<StudentRaitingDto, StudentRaitingOutputModel>()
+                .ForMember(dest => dest.RaitingType, opt => opt.MapFrom(src => new RaitingTypeOutputModel 
```

Comment:
you can register a mapping from StudentRaitingDto into RaitingTypeOutputModel and then this code will not be needed